apps/user-portal/src/components/account-recovery-security-qs.tsx

```diff
@@ -89,11 +91,11 @@ class SecurityQuestionsComponentInner extends React.Component<ComponentProps, an
      * question set ids of the questions fetched from the api.
      */
     public initModel = () => {
-        const {challenges, challengeQuestions} = this.state;
-        const challengesCopy = [{...challengeQuestions}];
+        const { challenges, challengeQuestions } = this.state;
```

Comment:
We only need to add a space between the curly brackets and the characters in the imports. Shall we remove this space in the constants declaration?